components/camel-azure/camel-azure-servicebus/src/main/java/org/apache/camel/component/azure/servicebus/client/ServiceBusClientFactory.java

```diff
@@ -79,9 +79,7 @@ private static ServiceBusClientBuilder.ServiceBusReceiverClientBuilder createBas
             final ServiceBusClientBuilder busClientBuilder, final ServiceBusConfiguration configuration) {
         final ServiceBusClientBuilder.ServiceBusReceiverClientBuilder receiverClientBuilder = busClientBuilder.receiver();
 
-        if (configuration.isDisableAutoComplete()) {
-            receiverClientBuilder.disableAutoComplete();
-        }
+        receiverClientBuilder.disableAutoComplete();
```

Comment:
Could you add a code comment here why this is disabled - eg that you take hands on commit / rollback in the consumer manually

Reply:
Done.